components/PolarChart.js

```diff
@@ -21,13 +21,13 @@ const TerpScoreChart = props => {
         label: "Terpene Scores",
         data: formatData(props, "score"),
         backgroundColor: [
-          "#5EB1BF",
-          "#54F2F2",
-          "#64403E",
-          "#F4E04D",
-          "#D2AB99",
-          "#7F7EFF",
-          "#00FFC5",
+          "rgba(250, 250, 110, 0.7)",
+          "rgba(170, 228, 121, 0.7)",
+          "rgba(100, 201, 135, 0.7)",
+          "rgba(35, 170, 143, 0.7)",
+          "rgba(0, 137, 138, 0.7)",
+          "rgba(23, 104, 119, 0.7)",
+          "rgba(42, 72, 88, 0.7)",
```

Comment:
when you try to assign them to the values in the terpene legend, the new colors for your chart seem less clear